<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RasterDisplayComposerDockWidgetBase</class>
 <widget class="QDockWidget" name="RasterDisplayComposerDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RasterDisplayComposer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1">
     <widget class="QPushButton" name="pushButton_load">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox_alpha"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="pushButton_save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_blue"/>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_noData">
      <property name="text">
       <string>Force no data</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Green band</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_red"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_green"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Red band</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Blue band</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBox_alpha">
      <property name="text">
       <string>Alpha channel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox_noData"/>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_bandName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
